internal/datastore/postgres/postgres_test.go

```diff
@@ -40,17 +41,36 @@ func (pgd *pgDatastore) ExampleRetryableError() error {
 	}
 }
 
+// the global OTel tracer is used everywhere, so we synchronize tests over a global test tracer
+var (
+	otelMutex         = sync.Mutex{}
+	testTraceProvider *trace.TracerProvider
+)
+
+func init() {
+	testTraceProvider = trace.NewTracerProvider(
+		trace.WithSampler(trace.AlwaysSample()),
+	)
+	otel.SetTracerProvider(testTraceProvider)
+}
+
 func TestPostgresDatastore(t *testing.T) {
+	t.Parallel()
+
 	for _, config := range []struct {
 		targetMigration string
```

Comment:
this is left around from the phased migration in 1.14, I believe. Can probably remove it now?